content/english/python-pixel/A.Colors/Pixels-on-Image/_index.md

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
Please change the path of the file to content/english/python-pixel/colors/pixels-on-image.md (content/english/python-pixel/A.Colors/Pixels-on-Image/_index.md)